courses/intro-to-spark/book/02_Flow_Analysis.rst

```diff
@@ -50,30 +50,30 @@ prior to entering the loop. As a consequence, the value read in the
 condition of the if statement may be uninitialized. Flow analysis will
 detect and report this error.
 
-.. code:: ada
+.. code-block:: ada
+
+    package Show_Uninitialized is
```

Comment:
Just for my understanding: isn't the `SPARK_Mode` needed here (and also in other examples below)?

Reply:
I explained why I did that on the ticket, using SPARK_Mode is redundant with the global configuration pragma that we use throughout (set up with the generated project file). I think it's cleaner to avoid adding SPARK_Mode everywhere.